packages/headless/src/features/case-assist/case-assist-state.ts

```diff
@@ -0,0 +1,11 @@
+export const getCaseAssistInitialState = (): CaseAssistState => ({
+  caseAssistId: 'CaseAssist',
```

Comment:
Is is fair to say that most clients won't actually have this as their default in their org ?

And so, that this would "randomly fail" when trying to request case assist for any random org ? Either you'd get a 4xx response from the platform, or, even worse, no document recommendation.

It's generally better to fail upfront when someone is doing the config, versus to have them debug something very cryptic and figure out that they need to pass in a case assist id when doing the engine configuration, and that it's not really an optional value after all.

Reply:
It is 100% impossible that this will be their ID. Let's fail it right off the bad then.